lnrpc/lightning.proto

```diff
@@ -2911,12 +2914,21 @@ message QueryRoutesRequest {
     */
```

Comment:
nit: maybe add a comment that they are not allowed when a blinded path is selected?